app/static/img/favicon/manifest.json

```diff
@@ -1,41 +1,45 @@
 {
- "name": "App",
+ "name": "Whoogle Search",
+ "short_name": "Whoogle",
+ "display": "fullscreen",
+ "scope": "/",
+ "start_url": "$APP_ROOT/templates/index.html",
```

Comment:
Is this necessary? I don't think `APP_ROOT` is actually being used here, and it looks like everything else should work without a `start_url` property.

```suggestion
```

Reply:
I was looking with the sample https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Installable_PWAs
Yes, Work with `start_url` as well.